deployment/development/docker-compose.yml

```diff
@@ -4,31 +4,38 @@ services:
   web:
     build:
       context: ../../packages
-      dockerfile: Dockerfile.web
+      dockerfile: web/Dockerfile
```

Comment:
this move serves two purposes:
 - Dockerfile is now along side the project that will be built with that dockerfile
   - each has their own .dockerignore
 - My Editor now correctly chooses the Dockerfile syntax :-)